app/config/config.yml

```diff
@@ -39,7 +39,7 @@ twig:
 # Doctrine Configuration
 doctrine:
     dbal:
-        driver:   pdo_mysql
+        driver:   "%database_driver%"
```

Comment:
I think this is a bad idea as having two environment use different DB drivers sounds scary.